packages/common/src/schemas/json/lightdash-dbt-2.0.json

```diff
@@ -76,6 +76,24 @@
                                 "type": "string",
                                 "enum": ["index", "label"]
                             },
+                            "group_details": {
+                                "type": "object",
+                                "patternProperties": {
+                                    "^[a-zA-Z0-9_]+$": {
```

Comment:
Should it be `^[a-z0-9_]+$` as the `metrics`?

Reply:
let's go with this, yes!